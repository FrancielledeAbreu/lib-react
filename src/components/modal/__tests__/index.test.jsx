import React from 'react'
import Modal from '..'
import renderer from 'react-test-renderer'

describe('Modal Test', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Modal isOpen={false} children='Teste' />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  it('renders correctly true', () => {
    const tree = renderer.create(<Modal isOpen children='Teste' />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
