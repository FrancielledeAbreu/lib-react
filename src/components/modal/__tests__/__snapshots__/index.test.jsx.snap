// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Test renders correctly 1`] = `<div />`;

exports[`Modal Test renders correctly true 1`] = `
<div>
  <div
    className="sc-gtssRu lRJPQ"
  >
    <div
      className="sc-bdnylx biflkL"
    >
      Teste
    </div>
  </div>
</div>
`;
